train_rl4lm.py

```diff
@@ -51,21 +46,19 @@
   policy:
     id: causal_lm_actor_critic_policy
     args:
-      model_name: gpt2
```

Comment:
this was the mistake in the previous commit, gpt2 was not the model that was supposed to be loaded. Good catch